### ISSUE TITLE
Documentation updates and organization

### ISSUE THREAD

**Issue by McFateM** [mnylc/islandora_multi_importer#48]
I'd like to make some changes/additions to the documentation of this module but am not quite sure where to apply them most effectively?  I see that the GoogleAPI-WIP branch has a "docs" folder and I personally like that approach.  However, I believe there's improved documentation also lurking in one or more branches of the Islandora-Collaborarion-Group fork of this repository.  

I wonder if anyone listening here might be able to help organize (merge) the different bits of documentation we've accumulated to make future additions/updates easier to post?  Or, can anyone suggest an effective workflow for handling documentation given the current state of this project?

**Comment by DiegoPino** [comment]
@McFateM i would say
1.- use the docs folder for a longer description.
2.- Use the main readme file as a way of summarizing and linking to the docs folder.

**Comment by McFateM** [comment]
Agreed.  I'm just not comfortable enough with Git or Github to know how we get there most effectively?  

Personally, I would love to see the Google* branches merged to Master, if possible, and that would presumably bring the "docs" folder structure with it.  Then follow your suggestion from there for making documentation changes or adding additional docs to that folder.

**Comment by DiegoPino** [comment]
@McFateM happening this week, i promise. I just need to clean some code and add a hidden functionality that has been on my desk for weeks, and well, today is Islandora Release and i'm manager! =)

**Comment by McFateM** [comment]
Understood.  I don't want to rush anyone, least of all @DiegoPino!  

I'm going to make a quick change to my hook_form_alter and will document that in the islandora_multi_importer.api.php file, then I need to move off to some ISLE work for a day or two.  Take care.

**Comment by McFateM** [comment]
@DiegoPino   Still hoping we have a mainstream docs and docs/images folders soon so I can add some documentation.  No big rush though... I really do need to focus on ISLE tomorrow.  Thanks.
